VirtoCommerce.Platform.Core/ModuleFileCollector/StyleCollector.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using VirtoCommerce.Platform.Core.Modularity;
+using VirtoCommerce.Platform.Core.VersionProvider;
+
+namespace VirtoCommerce.Platform.Core.ModuleFileCollector
+{
+    public class StyleCollector : FileCollector, IStyleCollector
```

Comment:
This class looks almost identical to the `ScriptCollector`. This can probably be improved.